package.json

```
@@ -52,7 +52,7 @@
     "lint-staged": "lint-staged",
     "next-with-deps": "./scripts/next-with-deps.sh",
     "next": "cross-env NEXT_TELEMETRY_DISABLED=1 node --trace-deprecation --enable-source-maps packages/next/dist/bin/next",
-    "next-no-sourcemaps": "cross-env NEXT_TELEMETRY_DISABLED=1 node --trace-deprecation packages/next/dist/bin/next",
+    "next-no-sourcemaps": "echo 'No longer supported. Use `pnpm next --disable-source-maps` instead'; exit 1;",
```

The suggested command also does not work though.

---

Right, because we override it in our monorepo. It would work for users though. I'll just remove the suggestion for now until we need it frequently. Can always be worked around with disabling our custom `next` override.

```suggestion
    "next-no-sourcemaps": "echo 'No longer supported.'; exit 1;",
```

---

This should do it: [`6cec9a7` (#71820)](https://github.com/vercel/next.js/pull/71820/commits/6cec9a7a395af5c3278c10b140d0d43deecc1c84)